src/server/database/setupDB.js

```diff
@@ -1,48 +0,0 @@
-import r from './rethinkdriver';
-
-const databases = ['action', 'actionTesting'];
-
-const database = [
```

Comment:
ohhh now I see it, neat!
what are your thoughts on integration testing with the DB? Will we add back in a separate DB for tests or just mock it & unit test?